src/commands/capsule/tasks/start/system.ts

```diff
@@ -0,0 +1,82 @@
+import * as fs from 'fs';
+import { integer, string } from '../../../../lib/flags'
+import debugFn = require('debug')
+import { appendTag, createTask } from '../../../../lib/tasks'
+import { NewTask } from '../../../../lib/api'
+import { Command } from '../../../../lib/command'
+
+const debug = debugFn(`capsule:tasks:start:system`)
+
+export default class TasksStartSystemCommand extends Command {
+
+  static description = `Start a system task with the given configuration`
+  static hidden = true
+
+  static flags = { // how to change ordering of these in help command?
+    task_type_name: string({
```

Comment:
Flags should not have underscores `_` in the name as it makes it awkward to type: `--task_type_name`

Further, as much as possible, strive to reduce typing an unnecessary context in flag names. For instance, `--task_type_name` in the context of a task command can simply be `--type`.

I'd rename, and be consistent across all task commands, to the following:

--type
--major or --version
--name
--assign-to
--file
--tag